content/collections/experiment-apis/en/experiment-management-api-experiments.md

```diff
@@ -1027,8 +1027,8 @@ Edit an experiment.
 
 |<div class="med-big-column">Name</div>|Requirement|Type|Description|
 |---|---|---|---|
-|`group_type`| Optional | string | Group type of the filter; can be null. Can be `User` value or one of the group values, eg `org _id`, `org name` |
-|`subprop_key`| Required | string | Filter's key; can be null. |
+|`group_type`| Optional | string | Group type of the filter; can be null. Can be `User` value or one of the group values, eg `org_id`, `org name` |
+|`subprop_key`| Required | string | Filter's key; cannot be null. |
```

Comment:
was this documented that the key can file null somewhere?

Reply:
I'll need to check. I don't remember documenting this as such.